api/envoy/extensions/filters/http/grpc_http1_reverse_bridge/v3/config.proto

```diff
@@ -30,6 +30,16 @@ message FilterConfig {
   // hide the gRPC semantics from the upstream, allowing it to receive and respond with a
   // simple binary encoded protobuf.
   bool withhold_grpc_frames = 2;
+
+  // When
+  // :ref:`envoy_api_field_extensions.http.grpc_http1_reverse_bridge.v3.FilterConfig.withhold_grpc_frames`
+  // is true, this option controls how envoy will calculate the upstream response size. When this
+  // is empty, envoy will buffer the upstream response to calculate its size. When this field is
+  // true, envoy can stream the response to the downstream and it will use the value of the response
+  // header with this name to set the `Content-Length` header. This allows the upstream to use
+  // chunked transfer encoding. If this option is specified, the upstream must set this header. If
+  // the header is repeated, only the first value will be used.
```

Comment:
I feel this whole explanation does not make sense to someone not deeply familiar with the context. Can you rewrite for an audience who does not have this?

Reply:
Thanks, I'll take another pass on the API comments tomorrow.